src/table/tableview/table-view.component.js

```diff
@@ -67,6 +68,7 @@ angular.module('patternfly.table').component('pfTableView', {
       // Need to deep watch changes in dtOptions and items
```

Comment:
There is no onToggleColvis in table-view.component.js file. It's in toolbar-component.js file. Additionally, I just followed the inputs/outputs styles described in [angular guide](https://docs.angularjs.org/guide/component#component-based-application-architecture). Is it necessary to customize the callback mechanism by ourselves ?